src/nerve/frontends/tui/commander/rendering.py

```
@@ -0,0 +1,346 @@
+        console.print("  [dim]No nodes in session[/]")
+    else:
+        for node_id, node_type in nodes.items():
+            console.print(f"  [bold]{node_id}[/] ({node_type})")
+    console.print()
+
+
+def print_timeline(
+    console: Console,
+    timeline: Timeline,
+    current_world: str | None,
+    nodes: dict[str, str],
```

Ruff (0.14.10)

Unused function argument: `nodes`

(ARG001)